packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -996,7 +996,15 @@ export default class Meetings extends WebexPlugin {
   fetchUserPreferredWebexSite() {
     return this.request.getMeetingPreferences().then((res) => {
       if (res) {
-        this.preferredWebexSite = MeetingsUtil.parseDefaultSiteFromMeetingPreferences(res);
+        const preferredWebexSite = MeetingsUtil.parseDefaultSiteFromMeetingPreferences(res);
+        this.preferredWebexSite = preferredWebexSite;
+        // @ts-ignore
+        this.webex.internal.services._getCatalog().setAllowedDomains(
+          // @ts-ignore
+          union(this.webex.internal.services._getCatalog().getAllowedDomains(), [
```

Comment:
this is a tiny optimization (probably overkill) so that the list of allowedDomains is unique. trying to prevent pushing the same domain again fro some reason. honestly it's probably overkill, because we don't expect this to get very big. i also considered converting the array to a Set() but ampersand's support of Set seems a little dodgy